core/dutydb/memory.go

```diff
@@ -16,14 +16,22 @@ import (
 	"github.com/obolnetwork/charon/core"
 )
 
+const (
+	attV1 = iota + 1
+	attV2 = iota + 1
```

Comment:
Hmm, this looks weird. I used to see just single `iota`, or we need to increment the second one?

Reply:
Yes, this is better probably, changed it.